CHANGES.rst

```diff
@@ -81,6 +89,26 @@ API changes
 
 - ``astropy.nddata``
 
+  - ``NDDataBase`` only defines an abstract ``uncertainty`` getter instead of
```

Comment:
How about combining this and the next one into one, phrased something like: ``NDDataBase`` does not implement a setter or getter for ``uncertainty``, which is now an `abstractproperty`.

And add the PR # in brackets.